src/main/java/com/contentgrid/gateway/runtime/RuntimeConfiguration.java

```diff
@@ -33,10 +33,12 @@
 import com.contentgrid.gateway.security.jwt.issuer.LocallyIssuedJwtGatewayFilter;
 import com.contentgrid.gateway.security.jwt.issuer.SignedJwtIssuer;
 import com.contentgrid.gateway.security.oidc.ReactiveClientRegistrationIdResolver;
+import com.contentgrid.thunx.encoding.json.ExpressionJsonConverter;
 import com.contentgrid.thunx.pdp.opa.OpaInputProvider;
 import com.contentgrid.thunx.pdp.opa.OpaQueryProvider;
 import com.contentgrid.thunx.spring.gateway.filter.AbacGatewayFilterFactory;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import java.nio.charset.StandardCharsets;
```

Comment:
Where is StandardCharsets used?

Reply:
It is not anymore; it was used in a previous iteration of the code.